posts/gov_press_release.md

```diff
@@ -0,0 +1,18 @@
+# Made Tech Awarded 3 Government Publishing Tenders
+
+We're excited to announce that we've been chosen to deliver three projects on the GOV.UK platform. Over the next few months, we'll be working closely with the GDS teams to deliver:
+
+**End To End Testing for Publishing Applications**
+GDS have built a multitude of microservices within the GOV.UK platform, each of which are thoroughly tested. An improvement they're keen to make is to have a similar level of testing on a broader level. We'll be working with them to provide a set of end to end tests for six of their publishing applications, ensuring the most common routes through and between each microservice are well tested and perform as expected.
```

Comment:
"between each microservice are well tested and perform as expected"

could maybe be something like:

"between each microservice work when integrated"

i.e. we know the apps work in isolation, but this is the confidence they work when glued together